code/src/components/Details.css

```diff
@@ -0,0 +1,71 @@
+.details-page {
```

Comment:
Nice job using separate .css files for each component. Basically styled components right 😄👍🏼